pkg/config/viper_test.go

```diff
@@ -47,6 +48,7 @@ func TestConcurrencyUnmarshalling(t *testing.T) {
 	config.SetDefault("baz", "test")
 
 	var wg sync.WaitGroup
+	errs := make(chan error, 100)
```

Comment:
Since we're running 1000 times in the example we could make this buffer size 1k too.

Reply:
Done. I’ve increased the chan size.